DOC_LOCAL_SETUP.md

```diff
@@ -6,4 +6,4 @@
 
 For more information on Github-Pages, see: https://help.github.com/categories/github-pages-basics/
 
-For more information about Jekyll, see: https://jekyllrb.com/docs/home/
\ No newline at end of file
+For more information about Jekyll, see: https://jekyllrb.com/docs/home/
```

Comment:
It would be nice to provide more detailed information how to use this documentation repository (developers documentation). So more or less:
- where content for documentation is located (_include/...)
- how documentation is build (_data/...)
- how to modify style (sass, compass, etc. -> at least links to some documentation), how to build it before pushing, etc.
- any other things that are required in order to make some changes in documentation project